mediathread/assetmgr/models.py

```diff
@@ -339,3 +339,22 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField(blank=True, null=True)
+    spine = models.ForeignKey(Source, blank=True, null=True)
```

Comment:
linking to a Sherdnote here would fit better with the current architecture, and make this more akin to a straight Asset.